# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/MotionPathUtils.cpp
# Commit: 473ebe5ec7aa
# Full Hash: 473ebe5ec7aa4f1c5da45ed580fb56258737a210
# Author: Boris Chiou <boris.chiou@gmail.com>
# Date: 2023-08-23 04:21:33
# Regressor Bug: 1819464
# File Overlap Count: 2
# Description:
#   Bug 1819464 - Part 2: Add content-box and stroke-box into transform-box in style. r=emilio
#   
#   Also, add a prefernce for content-box and stroke-box. So we have to
#   define a new type, TransformBox, and let content-box and stroke-box
#   behind the preference.
# ==============================================================================

diff -r 9c22e16e0567 -r 473ebe5ec7aa layout/base/MotionPathUtils.cpp
--- a/layout/base/MotionPathUtils.cpp	Tue Aug 22 19:49:03 2023 +0000
+++ b/layout/base/MotionPathUtils.cpp	Tue Aug 22 19:49:03 2023 +0000
@@ -33,8 +33,8 @@
   }
 
   auto transformBox = aFrame.StyleDisplay()->mTransformBox;
-  if (transformBox == StyleGeometryBox::ViewBox ||
-      transformBox == StyleGeometryBox::BorderBox) {
+  if (transformBox == StyleTransformBox::ViewBox ||
+      transformBox == StyleTransformBox::BorderBox) {
     return {};
   }
 